### ISSUE TITLE
Amazing prank my guy 10/10

### ISSUE THREAD

**Issue by Tqtann** [abhijithgoodboi/universities#1]
My professor at college sent me this link and I opened it knowing it was some sort of meme website since its a github website project.
The rickroll was amazing and perfect. 10/10

**Comment by abhijithgoodboi** [comment]
Can you tell where are you from and how did your professor get this link and btw I jst created this link to prank my frnds about the upcoming University result.

**Comment by Tqtann** [comment]
Coimbatore. Honestly I have no idea how he got that link but deleted it as soon as he sent it, I was lucky to see it quickly.

**Comment by abhijithgoodboi** [comment]
Coimbatore...! Are you under the bharathiar University? If yes that's why he share the link maybe.... Without realising it's fake one

**Comment by Tqtann** [comment]
yep, studying at an affiliated college under bharathiar university. yea he might have sent it without realising it. 
Either way it was a good prank

**Comment by abhijithgoodboi** [comment]
I just think to prank my friends that they are eagerly waiting and tensed of the result and i got what they want but I didn't think about it will goes out my class and campus

**Comment by Tqtann** [comment]
it got very popular in a small amount of time, but i hope there wont be any problems. yea same here with my friends asking about the results.

**Comment by abhijithgoodboi** [comment]
Btw my name is abhijith studying 2nd year BSc CS in AJK. what about you?

**Comment by Tqtann** [comment]
My names Vijay. Im doing bcom pa same year as you
